tests/Unit/Presentation/GreetingGeneratorTest.php

```diff
@@ -60,6 +60,29 @@ public function informalGreetingProvider(): array {
 		];
 	}
 
+	public function testGivenNoLastnameForInformalLastnameGreeting_neutralGreetingIsGenerated(): void {
+		$this->assertSame( 'mail_introduction_generic', $this->getGreetingGenerator()->createInformalLastnameGreeting( 'Herr', '' ) );
```

Comment:
These test lines could use some line breaks, e.g. expected and actual each on its own line.